### PR TITLE
Android: Some simple Clang-Tidy suggestions

### PR DESCRIPTION
**Pull request by Ebola16**
Split from #7252

### CODE REVIEW

File: Source/Android/jni/MainAndroid.cpp [hunk 1]
```diff
@@ -157,7 +157,7 @@ static bool MsgAlert(const char* caption, const char* text, bool yes_no, Common:
   return result != JNI_FALSE;
 }
 
-static void ReportSend(std::string endpoint, std::string report)
+static void ReportSend(const std::string& endpoint, std::string report)
```
Comment (JosJuice) [inline Source/Android/jni/MainAndroid.cpp:160]:
```suggestion
static void ReportSend(const std::string& endpoint, const std::string& report)
```

**Comment by delroth** [comment]
I don't have any objection against using auto when the right side of the assignment already repeats the type (which I imagine is what clang-tidy is finding here).

**Comment by Ebola16** [comment]
@dolphin-emu-bot rebuild

**Comment by Ebola16** [comment]
I'm somewhat indifferent to the `auto` changes but I like less complaints from Clang-Tidy.

**Pull request merged**